### PR TITLE
Fix footer links

### PR DESCRIPTION
**Pull request by Gristof**

### CODE REVIEW

**Comment by BirgitBader** [comment]
@Gristof Thanks!
For the AGB, Impressum and Datenschutz > is there a reason why this is German text and also linked to German pages, while Release Notes is English, but linked to a German page? Is it possible to have the whole footer consistently in English? (Would also apply for Kontakt > Contact then).

**Comment by Gristof** [comment]
It is possible. I just left it as it was.

**Comment by BirgitBader** [comment]
Ok. Can you change it?

**Comment by Gristof** [comment]
Will do

File: _includes/footer.html [hunk 1]
```diff
@@ -33,13 +33,13 @@
...
         <br>
-        <a href="https://www.epages.com/de/kontakt/">Kontakt</a> ·
+        <a href="https://www.epages.com/contact/">Contact</a> ·
         <a target="_blank" href="http://epages.com">epages.com</a> ·
         <a href="mailto:info@epages.com">info@epages.com</a> ·
-        <a href="https://www.epages.com/de/agb/">AGB</a> ·
-        <a href="https://www.epages.com/de/impressum/">Impressum</a> ·
-        <a href="https://www.epages.com/de/datenschutz/">Datenschutz</a> ·
-        <a href="https://www.epages.com/de/release-notes/">Release Notes</a>
+        <a href="https://www.epages.com/terms-conditions/">Terms and Conditions</a> ·
```
Comment (BirgitBader) [inline _includes/footer.html]: Terms & Conditions

**Pull request merged**